### DISCUSSION TITLE
Is Act of Remuneration intended to cover acts of compensating contractors?

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [CommonCoreOntology/CommonCoreOntologies#727] [Scope Questions]
I hope so, because I'd like to use it that way. (I'd also assume so, since that fits the standard use of "remuneration".) However, if it is, I think there's a problem with the current definition:

> An Act of Purchasing wherein a Financial Instrument is used by an Agent (the Employer) to compensate another Agent (the Employee) for the services they perform for the Employer.

The problem is that the definition restricts acts of remuneration to acts occurring in the context of an _employment relationship_, and CCO elsewhere distinguishes sharply between employment relationships and contractee/contractor relationships (see the scope note for Contractor Role).

**Comment by BrendaBraitling** [comment]
@gregfowlerphd 
I had to look this up, but I believe the definition is accurate as stated.  
I am not a professional ontologist, but have decades of experience in dealing with contracts and regulatory compliance... Remuneration seems to have a general "dictionary" definition of payment or compensation - which adds to the confusion.  However, in general use - in the world of business, regulation and taxation - it seems to require an Employer- Employee relationship and refers to compensation received by the **Employee - for Human Labor as a Person -** from the Employer.  See some links/excerpts below.

Contractors are party to an Agreement and are compensated based upon the terms of the Agreement alone. In the real world, it is handled as a business-to-business (or party-to-party) interaction.   Some tax regulations, contractual terms or grant stipulations require the contractor to handle payments to Person/Humans for "Acts of Human Labor" in a specified manner.  **So a Contractor seems to  not be a "Person" at all, but is a business organization** which is paid according to a contract and then carries out its own payment of Persons.  

As an aside... I am a little concerned about the term **"purchasing"** because employment is an internal activity within an organization. Purchasing is usually a business activity that is external to an organization.  Very different for accounting purposes... And - I personally feel that employees are compensated (paid) for services rendered and not purchased....🤔

[Investopedia](https://www.investopedia.com/terms/r/remuneration.asp#toc-what-is-remuneration)
"What Is Remuneration?
**Remuneration is the total compensation received by an employee** including base salary, bonuses, commission payments, overtime pay, and any other monetary benefits that the employee receives...

[UN - International Labour Organization 1951]( https://normlex.ilo.org/dyn/nrmlx_en/f?p=NORMLEXPUB:12100:0::NO::P12100_Ilo_Code:C100#:~:text=(a)%20the%20term%20remuneration%20includes,2.)
" Equal Remuneration Convention, 1951:  Article 1
For the purpose of this Convention--
(a) **the term remuneration includes the ordinary, basic or minimum wage or salary and any additional emoluments whatsoever payable directly or indirectly, whether in cash or in kind, by the employer to the worker and arising out of the worker's employment;**

**Comment by gregfowlerphd** [comment]
@BrendaBraitling: Thanks for the reply, Brenda! A few thoughts:

- I'm certainly happy to defer to your experience in this area, particularly given the links you provided. However, if CCO's use of 'remuneration' is meant to capture the way that term is used in a business context rather than its "dictionary definition", I'm inclined to think it would be helpful to add a rdfs:comment specifying this.
- The Wikipedia entry CCO cites as its definition source for Act of Remuneration _does_ specify that such an act must occur in the context of an employment relationship. However, that Wikipedia entry itself cites [this webpage](https://www.wordreference.com/definition/remuneration), where the definitions given also cover acts of compensating contractors. So I'm inclined to think the Wikipedia entry isn't a good definition source if the intent is to capture the "business context" use of 'remuneration'. (That said, there has been [some discussion](https://github.com/CommonCoreOntology/CommonCoreOntologies/discussions/370) on here about moving away from Wikipedia as a definition source, so this bullet point might soon be rendered moot.)
- I think Act of Purchasing, as it's defined in CCO, does cover employee compensation. It might be worth considering whether that definition should be modified in light of your point, however.

**Comment by BrendaBraitling** [comment]
@gregfowlerphd You bring up two very good points.  Sources should be authoritative and we need to be cautious regarding details at midlevel.  As you state above, the definition source is probably being handled.  However the logic and semantic meaning need to be carefully considered in light of being universally useful to various domain ontologies and task ontologies as they link up to CCO. CCO branches need to be logically clear in their relations to one another and up to BFO.  Perhaps there is too much detail.

It is my understanding that applied ontologies on this level need to 'be specific enough' to 'be useful'  - and , ideally, provide a means for many domains to link up... Perhaps this delineation is too specific because it is getting into the domain of employment law terms and tasks of business accounting and operations.  

Perhaps CCO should terminate at the level of **business expense** which allows a more specific Business Domain ontology or Business Task ontology to split the fine hairs... At level of 'business income' and 'business expense,' CCO allows all types of business practices and tasks and all type of business organizations to link up to it - from multinational corporations and governments to nonprofit organizations and lemonade stands.  It also allows for localization to various cultures around the world regarding how business is done. 

This 'pruning' allows the CCO to BFO links and logic to be clear and the the link-up and logical connections to be clean from any intricate business domain ontology.  It is also expeditious in getting a robust, unified CCO to standardization.  I am inclined to think that there are several CCO issues that may be resolved by carefully contemplating the terminal elements of CCO's reach.  @johnbeve
